layers/+intl/japanese/packages.el

```diff
@@ -0,0 +1,93 @@
+;;; packages.el --- Japanese Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
```

Comment:
It's [OK to change](https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org#author-of-a-new-layer) author name to yours name 😸